bin.src/processFile.py

```diff
@@ -323,12 +328,13 @@ def __iter__(self):
                         help="display the specified information to stdout and quit (unless run is specified).")
     parser.add_argument("-L", "--loglevel", help="logging level", default="WARN",
                         choices=CaseInsensitiveChoices('DEBUG', 'INFO', 'WARN', 'FATAL'))
-    parser.add_argument("-T", "--trace", nargs="*", action=pbArgparse.TraceLevelAction,
-                        help="trace level for component", metavar="COMPONENT=LEVEL")
+    if False:
+        parser.add_argument("-T", "--trace", nargs="*", action=pbArgparse.TraceLevelAction,
+                            help="trace level for component", metavar="COMPONENT=LEVEL")
 
     parser.add_argument('--debug', '-d', action="store_true", help="Load debug.py?", default=False)
     parser.add_argument('--display', nargs='*', help="Display sources on an image display",
-                        choices=CaseInsensitiveChoices('True', 'showApertures', "showShapes"),
+                        choices=CaseInsensitiveChoices('True', 'showApertures', "showShapes", "showPSFs"),
```

Comment:
Add 'showPSFs' option to documentation in `doc/main.dox`.

Actually, now that I look at it more closely, update documentation in `doc/main.dox` from `--ds9` to talk about the more general `--display` option.